cdap-app-fabric/src/main/java/io/cdap/cdap/internal/sysapp/SystemAppEnableExecutor.java

```diff
@@ -112,17 +112,14 @@ private ApplicationWithPrograms deploySystemApp(Arguments arguments) throws Exce
     ArtifactSummary artifactSummary = arguments.getArtifact();
 
     KerberosPrincipalId ownerPrincipalId =
-        arguments.getOwnerPrincipal() == null ? null
-            : new KerberosPrincipalId(arguments.getOwnerPrincipal());
+      arguments.getOwnerPrincipal() == null ? null : new KerberosPrincipalId(arguments.getOwnerPrincipal());
```

Comment:
here it's correct to do 2 space indent since it's not in the middle of a method call

Reply:
Oh ok. In this case the whole line came into 1 single line <120 chars so just made a one line.